allauth_2fa/app_settings.py

```diff
@@ -5,3 +5,5 @@
 TEMPLATE_EXTENSION = getattr(settings,
                              'ALLAUTH_2FA_TEMPLATE_EXTENSION',
                              allauth_settings.TEMPLATE_EXTENSION)
+
+ALWAYS_REVEAL_BACKUP_TOKENS = bool(getattr(settings, 'ALLAUTH_2FA_ALWAYS_REVEAL_BACKUP_TOKENS', True))
```

Comment:
I don't think this `bool` cast will generally do what is expected with strings. Is this necessary here? (E.g. `bool('False') == True`)

Reply:
OK. It probably isn't any worse than the implicit conversion that will happen when this is used in the template anyway.